### PR TITLE
Format Travis output

### PR DESCRIPTION
**Pull request by bcmn**
The current output from Travis jobs is >800 lines for master. This PR adds [xcpretty-travis-formatter](https://github.com/kattrali/xcpretty-travis-formatter) to enable folding of the Build & Test phases, & whilst still presenting any failures at the top level, to improve quick readability.

### CODE REVIEW

**Comment by allenhumphreys** [comment]
@bcmn We're also leveraging xcpretty output for the Carthage bootstrapping, its in `bootstrap.sh`, in order to get some more verbose output for the purpose of inspecting CI builds. Though thinking about it now, I'm not sure if it's necessary anymore, I had it on to help debug the CI setup earlier.

**Comment by bcmn** [comment]
Yeah, I realised I'd forgotten about bootstrap pretty quickly 😝 This should fold that block, but I'm not sure it'll surface errors in there in the same way 😕 Irritating...

File: .travis.yml [hunk 1]
```diff
@@ -4,5 +4,7 @@ script:
   - xcodebuild -version
   - carthage version
   - swiftlint version
+  - echo 'Bootstrap' && echo -en 'travis_fold:start:Bootstrap\\r'
   - "./bootstrap.sh"
-  - set -o pipefail && env "NSUnbufferedIO=YES" xcodebuild test -scheme WWDC -project WWDC.xcodeproj | xcpretty
+  - echo -en 'travis_fold:start:Bootstrap\\r'
```
Comment (allenhumphreys) [inline .travis.yml]: end?

**Comment by allenhumphreys** [comment]
We have a `Gemfile.lock` file but no `Gemfile`, is that right?

**Comment by bcmn** [comment]
Ahaha. You're missing my pedantic streak, that folding thing keeps getting detached from the colourised "Bootstrap" line 😑 Shell formatting is hard 😛

**Comment by bcmn** [comment]
Hm, I've not had much luck with the Slack integration. It wasn't super connected to this PR anyway so I'll probably just revert & merge.

**Pull request merged**